project/http/requests/useragent/userAgent.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+class UserAgent:
+
+    def __init__(self, web_proxy_list=[]):
+        self.agent_file = '../data/user_agents.txt'
+        self.useragents = self.load_user_agents(self.agent_file)
+
+    def load_user_agents(self, useragentsfile):
+        """
+        useragentfile : string
+            path to text file of user agents, one per line
+        """
+        useragents = []
+        with open(useragentsfile, 'rb') as uaf:
+            for ua in uaf.readlines():
+                if ua:
+                    useragents.append(ua.strip()[1:-1-1])
+        return useragents
+
+    def random_user_agent(self):
```

Comment:
I would rename the method to something more descriptive like: get_random_user_agent